platform/android/java/gradle.properties

```diff
@@ -9,6 +9,8 @@
 
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
+android.enableJetifier=true
+android.useAndroidX=true
```

Comment:
The comments above doesn't match these two settings. Move them to another location.

Reply:
Should I remove most of the comments in this file? they feel very boilerplaty (exept the one above of course)